src/lido/LidoOracle.sol

```diff
@@ -0,0 +1,53 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.20;
+
+import {IERC20} from "../interfaces/IERC20.sol";
```

Comment:
We can use the IERC20 interface in forge-std. This way the local one can be removed.

import "forge-std/interfaces/IERC20.sol";